articles/howtos/create-new-decorator.md

```diff
@@ -0,0 +1,222 @@
+---
+uid: CreateNewDecorator
+---
+# How can I create a new Decorator?
+
+Click [here](xref:Decorator) to find out more about Decorators.
+
+This how-to will serve as an example of how to create a new Decorator that will inject code at runtime for a Template that has been setup for it.
+
+This how-to will build up from the [previous](xref:MakeTemplateDecoratorReady) how-to, which showed how to make a Template ready for Decorators. Please interpolate on your end what is written here to make the necessary adjustments on what you're trying to achieve.
+
```

Comment:
Maybe we just omit this to keep the article as concise as possible.
```suggestion
```

Reply:
I guess, the whole point of a howto is to be stand-alone. I think you should be able to execute both how-tos in any order. So we could take it out.

Reply:
This relates to the comment below, we'll address both together I think.

Reply:
Again referring to the docs guide article:

> # LEAVE THINGS OUT
> Practical usability is more valuable than completeness. Tutorials need to be complete, end-to-end guides; how-to guides do not. They can start and end where it seems appropriate to you. They don’t need to mention everything that there is to mention either, just because it is related to the topic. A bloated how-to guide doesn’t help the user get speedily to their solution.

With that in mind, I think we remove this line, let me know what you think. If you're okay with that go ahead and make the change, otherwise let me know and I'll do it.